src/showtasklist.js

```diff
@@ -3,6 +3,9 @@ import iconTrash from './images/icon-trash.svg';
 import deleteTask from './deletetask.js';
 import editTask from './edittask.js';
 import checkTask from './checktask.js';
```

Comment:
- [optional] You've done a great job I must say, I observed that you've implemented the `check status task` in previous milestone which shouldn't be. 👍 **Kindly take note for subsequent PRs.**
- [ ] Please kindly implement this functionality as well according to the project requirement. 

<img width="556" alt="Untitled" src="https://github.com/jicamargo/todo/assets/103386183/862a061e-e4ae-4d51-b62e-687b3c259a16">